### PR TITLE
Emergency tab button: style update 

### PR DESCRIPTION
**Pull request by kiwansim**
Update:

Simulator Screen Shot - iPhone 11 - 2020-07-07 at 23 32 44

------

Done: 
- Emergency button matches Zeplin 

Questions:
- Unsure if absolute positioning is best practice. Look for other hacky implementations please! 
- To match Zeplin completely, the tab bar icons need to all be shifted to the right a bit. Is that something we want? 

Screen Shot 2020-07-03 at 11 14 04 AM

### CODE REVIEW

File: js/screens/EmergencyScreen.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable global-require */
+import React from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+
+import Constants from 'expo-constants';
+
+export default function Alerts() {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>{'This is the Emergency pop-up!'}</Text>
```
Comment (JacobJaffe) [inline js/screens/EmergencyScreen.js:10]: Can we throw this in the localization file so we don't lose this string?

File: js/screens/EmergencyScreen.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable global-require */
```
Comment (JacobJaffe) [inline js/screens/EmergencyScreen.js:1]: Is this needed here?

File: js/navigation/MainTabs.js [hunk 3]
```diff
@@ -70,6 +73,49 @@ const MainTabs = () => {
           ),
         }}
       />
+      <Tabs.Screen
+        name={routes.main.emergency}
+        component={Emergency}
+        options={{
+          // eslint-disable-next-line react/prop-types
```
Comment (JacobJaffe) [inline js/navigation/MainTabs.js:80]: I think this isn't need because of this on the file

File: js/navigation/MainTabs.js [hunk 4]
```diff
@@ -70,6 +73,49 @@ const MainTabs = () => {
...
+          // eslint-disable-next-line no-unused-vars
+          tabBarIcon: ({ color, size }) => (
+            <View
+              style={{
+                position: 'absolute',
+                bottom: 2, // space from bottombar
+                height: 64,
+                width: 64,
+                borderRadius: 100,
+                backgroundColor: '#FFF4ED',
```
Comment (JacobJaffe) [inline js/navigation/MainTabs.js:91]: Can the colors pull from `styles/colors` to ensure we align with Zeplin theming?

File: js/screens/EmergencyScreen.js (hunk 1 above)
Comment (kiwansim) [reply js/screens/EmergencyScreen.js:10]: It's only a temporary placeholder until the emergency modal is linked to this, so is adding it to localization file necessary?

Comment (JacobJaffe) [reply js/screens/EmergencyScreen.js:10]: Oo! Good point. I misread this as placeholder text for different text

**Pull request merged**